scripts/west_commands/build.py

```diff
@@ -137,6 +137,8 @@ def do_add_parser(self, parser_adder):
                             application.''')
         parser.add_argument('-c', '--cmake', action='store_true',
                             help='Force CMake to run')
+        parser.add_argument('--just-cmake', action='store_true',
```

Comment:
I would prefer `--cmake-only`. I think that's the usual way of restricting the scope of the execution of a command-line tool, although I might be biased by the ones I use.

Reply:
Sure, np